avocado_vt/plugins/vt_joblock.py

```diff
@@ -105,6 +105,11 @@ def pre(self, job):
                 self._lock(job)
         except Exception as detail:
             msg = "Failure trying to set Avocado-VT job lock: %s" % detail
+            files = list(self._get_lock_files())
```

Comment:
Oups, that didn't occur to me. How about `try/except files = None` to log at least the lockdir in case it can't obtain the lockfiles?